src/RazorSdk/Targets/Sdk.Razor.CurrentVersion.targets

```diff
@@ -50,9 +50,11 @@ Copyright (c) .NET Foundation. All rights reserved.
         <_TargetingNET50OrLater>true</_TargetingNET50OrLater>
         <_TargetingNET60OrLater>true</_TargetingNET60OrLater>
         <UseRazorSourceGenerator Condition="'$(UseRazorSourceGenerator)' == '' ">true</UseRazorSourceGenerator>
+        <!-- The incremental source generator API is disabled on VS 16.11 and below so we disable it there. -->
+        <UseRazorSourceGenerator Condition="'$(VisualStudioVersion)' &lt; '17.0'">false</UseRazorSourceGenerator>
         <!-- Required to support incremental source generator -->
         <LangVersion Condition="'$(UseRazorSourceGenerator)' != '' ">preview</LangVersion>
```

Comment:
Can we remove this now? C# 10 is enabled by default for net6.0 projects.

Reply:
Yep!

Reply:
This configuration doesn't look like its propogating over to the incremental generator so it's failing to run. Reverting for now. Will ping compiler team to get a sense of when it is safe to remove this (or if maybe there is another thing keeping the generator from running).